eslint-plugin-expensify/tests/no-api-side-effect-actions.test.js

```diff
@@ -0,0 +1,86 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../no-api-side-effect-actions');
+const message = require('../CONST').MESSAGE.NO_API_SIDE_EFFECTS_ACTIONS;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('no-api-side-effect-actions', rule, {
+    valid: [
+        {
+            code: `
+                function test() {
+                    API.write('Report_AddComment', params);
+                }
+            `,
+            filename: './src/libs/actions/Test.js',
+        },
+        {
+            code: `
+                import Action from './src/libs/actions/Action.js'
+                function test() {
+                    Action.perform(params);
+                }
+            `,
+            filename: './src/libs/actions/Test.js',
+        },
+        {
+            // Test that two functions in one file with different API/action calls are valid
```

Comment:
I love the comments like this where we describe what we're testing for in plain English, can we add these to all test cases?